<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralWizardPage</class>
 <widget class="QWizardPage" name="GeneralWizardPage">
  <property name="title">
   <string>General settings</string>
  </property>
  <property name="subTitle">
   <string>Adjust general settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Language:</string>
     </property>
     <property name="buddy">
      <cstring>langComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="langComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Font:</string>
     </property>
     <property name="buddy">
      <cstring>fontComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sizeSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="blockSpinBox">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Scrollback:</string>
     </property>
     <property name="buddy">
      <cstring>blockSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Messages:</string>
     </property>
     <property name="buddy">
      <cstring>timeStampCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="timeStampCheckBox">
     <property name="text">
      <string>Timestamp</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>langComboBox</tabstop>
  <tabstop>fontComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
